docs/acts_project.md

```diff
@@ -0,0 +1,36 @@
+# The ACTS project
+
+The *A Common Tracking Software (Acts)* project is an attempt to preserve and evolve the track reconstruction software of the LHC era towards HL-LHC and beyond. It has been initiated in 2016 starting from the [ATLAS Common Tracking Software](https://gitlab.cern.ch/atlas/athena/-/tree/master/Tracking). Given the changing computing landscape, dedicated care of parallel code execution is taken, and is written in `C++17`. 
+
+A [coherent write-up of the project](https://link.springer.com/article/10.1007/s41781-021-00078-8) has been published in 2022 in Springer's CSBS.
+
+```{note}
+Acts is designed as a library that *contains components* for assembling a track reconstruction suite for High Energy Physics and Nuclear Physics. Acts does not strive to provide a complete experiment framework, but rather modules and connections to be used within an experiment context. These connections contain e.g.~binding mechanisms to different geometry libraries, a cost-free yet flexible mechanism to use experiment specific contextual data (calibrations, detector alignment, experiment condidtions), or simply the possibility to integrate an external screen logging facility.
+```
+
+The library is structured as follows:
+ * The `Core` library contains considered production ready components that can be interfaced to experiment code
+ * The `Plugin` folder contains additional extensions that can be optionally switched on to use increase the functionality of the software suite, but also in general increase dependencies to other/thirdparty libraries
+ * The `Fatras` library contains a fast track simulation module, that is based on the same concepts that are used for the [ATLAS Fatras](https://cds.cern.ch/record/1091969?ln=de) fast track simulation  
+ * An `Examples` folder that contains a minimal test framework used for showcasing and integration testing, 
+ * A `Tests` folder that contains unit tests, benchmark tests and other integration tests
+
+
+```{tip} Developers and R&D meetings can be found in the [Acts indico category](https://indico.cern.ch/category/7968/).
```

Comment:
```suggestion
```{tip}
Developers and R&D meetings can be found in the [Acts indico category](https://indico.cern.ch/category/7968/).
```